Where_Is_My_Wife/Assets/_Main/Scripts/Story/StoryBoardAnimation.cs

```diff
@@ -29,11 +41,45 @@ private async UniTaskVoid StoryBoardAnimationAsync()
             for (int i = 0; i < comicImages.Length; i++)
             {
                 comicImages[i].DOFade(1, fadeDuration); 
-                await UniTask.Delay((int)((fadeDuration + intervalBetweenImages) * 1000));
+                await UniTask.Delay(TimeSpan.FromSeconds(fadeDuration + intervalBetweenImages));
             }
+            
+            ShowContinueButton();
+        }
+
+        private void ShowContinueButton()
+        {
+            _skipPrompt.SetActive(false);
             _nextSceneButton.gameObject.SetActive(true);
             _nextSceneButton.interactable = true;
             _nextSceneButton.Select();
         }
+
+        private void AdvanceSkipPrompt()
+        {
+            _submitPressCount++;
+            
+            if (_submitPressCount == 1)
+            {
+                _skipPrompt.SetActive(true);
+                return;
+            }
+            
+            SkipAsync().Forget();
+        }
+        
+        private async UniTaskVoid SkipAsync()
+        {
+            _skipPrompt.SetActive(false);
+            
+            foreach (var image in comicImages)
+            {
+                image.color = Color.white;
+            }
+
+            await UniTask.DelayFrame(1); // Prevent instant button press
+            
+            ShowContinueButton();
```

Comment:
Add this here.
InputEventManager.Instance.SubmitStartAction -= AdvanceSkipPrompt;